raiden-ts/src/helpers.ts

```diff
@@ -491,10 +490,7 @@ function validateDump(
  * @param deps.network - current network
  * @param deps.contractsInfo - current contracts
  * @param deps.log - Logger instance
- * @param storage - either [[Storage]] or [[RaidenState]] or
```

Comment:
`storage` line needs to stay (or be renamed, if you wish), since JSDocs requires that an object containing documented properties also have its `@param` entry.

Reply:
:heavy_check_mark: